core/src/marine_core.rs

```diff
@@ -37,6 +39,22 @@ pub struct MarineCore {
     modules: HashMap<String, MModule>,
 }
 
+pub struct OutdatedModule {
+    pub name: String,
+    pub version: Version,
+    pub stable_version: Version,
+}
+
+impl OutdatedModule {
+    pub fn new(name: String, version: Version, stable_version: Version) -> Self {
+        Self {
+            name,
+            version,
+            stable_version,
+        }
+    }
+}
+
```

Comment:
Suggest to put it to a separate file

Reply:
We decided to completely remove it, so I'll do it